### ISSUE TITLE
WorkItemFormService.(get/add)WorkItemRelations ignore comments

### ISSUE THREAD

**Issue by FlexiGit** [Microsoft/vss-web-extension-sdk#131]
When using the `WorkItemFormService` to get or add work item relations of the current work item the links will be retrieved or created alright. 
Unfortunately the service totally ignores the link comments. When retrieving the links the resulting `WorkItemRelation` objects do not contain the expected `attribute["comment"]` field, although I can see the comment on the form page.
Additionally when adding new links with the service the `attribute["comment"]` field is ignored again. 

When I use the `WorkItemTrackingHttpClient` to retrieve the links, the field is present and filled as expected. So this is a viable workaround for getting the comments.
For adding the links this is not possible, because this would cause major race conditions and update conflicts on the work item.

**Comment by sferg-msft** [comment]
You're correct, and I will get this fixed right away.  Thanks for reporting!

**Comment by FlexiGit** [comment]
First of all: Thanks for the prompt reaction! The add side is working now.

But we need a fix for the get side, too. My promoted workaround does not work, when I added a link through the `WorkItemFormService`, but have not yet saved the work item. The `WorkItemTrackingHttpClient` cannot see the link then, yet, and the `WorkItemFormService` sees the link, but not the comment.

**Comment by v-tsymbalistyi** [comment]
Hello!
Is there any progress on this?
I am building an extension for devops and need to be aware of comment updates.
Or is there some other decent way to do this?
